core/src/main/java/io/opencensus/stats/StatsManager.java

```diff
@@ -32,5 +32,5 @@
   /**
    * Returns the default {@link StatsContextFactory}.
    */
-  abstract StatsContextFactory getStatsContextFactory();
+  public abstract StatsContextFactory getStatsContextFactory();
```

Comment:
The users of the views and the users of `StatsContextFactory` are usually different. There isn't much benefit to bundle these two functionalities. I would keep this method as package-private so that `StatsManager` is only a view provider, so that the output component (`StatsManager`) and the input component (`StatsContextFactory`) can be faked and injected separately.